head.go

```diff
@@ -1427,6 +1427,8 @@ func (s *memSeries) truncateChunksBefore(mint int64) (removed int) {
 
 // append adds the sample (t, v) to the series.
 func (s *memSeries) append(t int64, v float64) (success, chunkCreated bool) {
+	// Based on Gorilla white papers this offers the best compression ratio
+	// so anything bigger that this doesn't offer any benefits.
```

Comment:
best -> near-optimal

doesn't offer any benefits -> has diminishing returns and increases the time range within which we have to decompress all samples